bmi_calculator/bmi_calculator.py

```diff
@@ -0,0 +1,129 @@
+import sys
+
+from PyQt6.QtWidgets import (
+    QApplication,
+    QDialog,
+    QWidget,
```

Comment:
Removed unused variables or module imports.  For example, you have imported  QWidget , but not used any where